Dockerfile

```diff
@@ -9,9 +9,10 @@ RUN apt-get update && apt-get install -y \
     gdal-bin \
     libproj-dev \
     libgeos-dev \
-    libspatialite-dev
+    libspatialite-dev \
+    git
 RUN pip3 install -U pip setuptools wheel
-RUN pip3 install -U .
+RUN git clone https://github.com/openwisp/openwisp-controller && cd openwisp-controller/ && pip3 install -U .
```

Comment:
I don't think cloning from `git` will work for our cause here.
It'll be getter if we use `COPY` to copy local repository, that will enable us to make changes in the repository and use/test it locally.